plugins/nextjs/src/pageGenerator/index.ts

```diff
@@ -17,7 +17,7 @@ export const pageGenerator: Generator = {
 
 async function generatePage(inputOptions: Options) {
   // Get workspaces
-  const cwd = process.cwd();
+  const cwd = inputOptions['parentDir'] as string;
```

Comment:
I think what we are doing here is giving the testing suites the ability to simulate running the generator in a different directory than the current working directory.  The users of the generator will always want to run it from the CWD so I think we should make inputOptions['parentDir'] default to CWD if it isn't defined.  That way the testing suite can override it but users aren't having to define it.

Reply:
making changes according to comments

Reply:
A commit has been added to address the comments